src/Libraries/RevitNodes/Application/Document.cs

```diff
@@ -62,6 +66,171 @@ public static Document Current
             get { return new Document(DocumentManager.Instance.CurrentDBDocument); }
         }
 
+        public void PurgeUnusedPostableCommand(bool deepPurge)
```

Comment:
this unused thing needs to be deleted

Reply:
🤦‍♂️.. yea forgot to delete my test node